tests/components/recorder/test_models.py

```diff
@@ -241,3 +242,16 @@ async def test_process_timestamp_to_utc_isoformat():
         == "2016-07-09T21:31:00+00:00"
     )
     assert process_timestamp_to_utc_isoformat(None) is None
+
+
+async def test_event_to_db_model():
+    """Test we can round trip Event conversion."""
+    event = ha.Event(
+        "state_changed", {"some": "attr"}, ha.EventOrigin.local, dt_util.utcnow()
+    )
+    native = Events.from_event(event).to_native()
+    assert native == event
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)